webapp/src/index.tsx

```diff
@@ -5,23 +5,40 @@ import type {Store, Action} from 'redux';
 
 import type {GlobalState} from '@mattermost/types/store';
 
-import MSTeamsAppManifestSetting from '@/components/admin_console/app_manifest_setting';
+import AppID from '@/components/admin_console_settings/app_id';
+import AppName from '@/components/admin_console_settings/app_name';
+import AppVersion from '@/components/admin_console_settings/app_version';
```

Comment:
Couldn't we use one single component for the three fields? These three are the same.

Reply:
I implemented this suggestion in the follow-up PR:  https://github.com/mattermost/mattermost-plugin-msteams-devsecops/pull/57